### ISSUE TITLE
Log functionality not working after Pimcore maintenance compress logs

### ISSUE THREAD

**Issue by marekgach** [dpfaffenbauer/ProcessManager#32]
| Q                | A
| ---------------- | -----
| Bug report?      | yes
| Feature request? | no
| BC Break report? | no
| RFC?             | no
| Branch?          | master

Method `Pimcore\Log\Maintenance::cleanupLogFiles()` which seems to be run during Pimcore maintenance is compressing the log files into gz and keeping them for 7 days. So as your library is storing the logs into `PIMCORE_LOG_DIRECTORY` those are affected as well. 

Your code ProcessManagerBundle\Logger\DefaultHandlerFactory is expecting for `getLog` method the file name is `process_manager_%s.log` however the name is `process_manager_%s.log.gz`. That is the reason that method will fail to load compressed log files and this lead to not working functionality to show or download log for log entries which were compressed. This is affecting than the controller method `ProcessManagerBundle\Controller\ProcessController::getLog` which is unable to return the log data.

I think it may be fine to amend `DefaultHandlerFactory::getLog` and `DefaultHandlerFactory::cleanup` to try to load+unzip / unlink the file with .gz if file with .log does not exist. Do you think there is some another place which needs to be changed? I can do the PR then.

**Comment by dpfaffenbauer** [comment]
I'd be great if you could provide a PR

**Comment by kubaplas** [comment]
@dpfaffenbauer shouldn't use of a different directory for logs be enough?

@wpeisert did something like this:
```yaml
    process_manager.default_handler_factory:
        class: ProcessManagerBundle\Logger\DefaultHandlerFactory
        public: true
        arguments:
            - '%kernel.logs_dir%/process'
````

**Comment by dpfaffenbauer** [comment]
I rather see that issue in Pimcore. Since `var/log` is the default directory for symfony, we should store our logs there as well

**Comment by marekgach** [comment]
@kubaplas Your solution is something I have now as a workaround.
@dpfaffenbauer I agree it should be under Symfony folder. We cannot rely on log rotation and other stuff is set correctly on server/Docker container.

I will prepare that PR in the upcoming days. I need to have that fix on our site before going live.

**Comment by dpfaffenbauer** [comment]
fixed in 2.6
